packages/rules-monorepo/src/ensure-boll.ts

```diff
@@ -0,0 +1,67 @@
+import { join, dirname } from "path";
+import fs from "fs";
+import { FileContext, RuleOptions } from "@boll/core";
+import chalk from "chalk";
+
+const exists = (path: string) => {
+  return new Promise<boolean>((resolve, reject) => {
+    fs.stat(path, err => {
+      if (err) {
+        resolve(false);
+      } else {
+        resolve(true);
+      }
+    });
+  });
+};
+
+/**
+ * Check to see if packages have a boll config, and a lint verb
+ * @param {*} logger
+ * @param {*} options
+ * @returns
+ */
+export const EnsureBoll = {
+  name: "EnsureBoll",
+  check: async (files: FileContext[], options: RuleOptions) => {
+    const errors: { formattedMessage: string; status: number }[] = [];
+
+    await Promise.all(
+      files.map(async ({ filename, content }) => {
+        const pkg = JSON.parse(content);
+
+        let hasBoll = false;
+        let hasIgnore = false;
+        let bollConfig = join(dirname(filename), ".boll.config.js");
```

Comment:
is .boll.config.js the only config name you can use?

Reply:
AFAIK, I'll check tho.